apps/s_server.c

```diff
@@ -2527,7 +2499,7 @@ static int init_ssl_connection(SSL *con)
 
     if (i <= 0) {
         if ((dtlslisten && i == 0)
-                || (!dtlslisten && retry)) {
+            || (!dtlslisten && retry)) {
```

Comment:
The original indentation was intentional. `if (` is exactly 4 chars and therefore the conditional and body merge into one and it is difficult to see where one stops and the next starts.